frontend/src/components/common/AdminUserTable.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TableContainer,
+  Text,
+} from "@chakra-ui/react";
+
+type AdminUser = {
+  firstName: string;
+  lastName: string;
+  email: string;
+};
+
+interface AdminUserTableProps {
+  adminUserArray: AdminUser[];
+}
+
+const AdminUserTable = ({
+  adminUserArray,
+}: AdminUserTableProps): React.ReactElement => {
+  return (
+    <TableContainer
+      padding="10px"
+      border="1px solid"
+      borderColor="grey.200"
+      borderRadius="12px"
+    >
+      <Table sx={{ tableLayout: "fixed" }} variant="unstyled" size="md">
+        <Thead>
+          <Tr>
+            <Th>
+              <Text textStyle="link" fontWeight="700" textTransform="none">
```

Comment:
i believe the fontWeight is handled in the text style
```suggestion
              <Text textStyle="link" textTransform="none">
```